src/poliastro/twobody/orbit.py

```diff
@@ -228,6 +228,38 @@ def from_vectors(cls, attractor, r, v, epoch=J2000, plane=Planes.EARTH_EQUATOR):
         ss = RVState(attractor, r, v)
         return cls(ss, epoch, plane)
 
+    @classmethod
+    def from_coord(cls, attractor, coord):
+        """Creates an `Orbit` from an attractor and astropy `SkyCoord` instance.
+        This method accepts `SkyCoord` in any reference frame unlike `Orbit.from_vector`
+        which can accept inputs in only inertial reference frame centred at attractor.
+        Also note that the frame information is lost after creation of the orbit and
+        only the inertial reference frame at body centre will be used for all purposes.
+        Parameters
+        ----------
+        attractor: Body
+            Main attractor
+        coord: astropy.coordinates.SkyCoord
+            Position and velocity vectors in any reference frame. Note that `SkyCoord` must have
```

Comment:
There is some error in this docstring: `sphinx.errors.SphinxWarning: /home/circleci/repo/venv/lib/python3.6/site-packages/poliastro/twobody/orbit.py:docstring of poliastro.twobody.orbit.Orbit.from_coord:9:Unexpected indentation.`

https://circleci.com/gh/poliastro/poliastro/1936

Please do

```
sphinx-build -vW -b html docs/source docs-html
```

locally to make sure it's fixed.